### DISCUSSION TITLE
Website is working in localhost and vercel but not in aws hosted live site.

### DISCUSSION THREAD

**Discussion by saiful-shaakil** [vercel/next.js#54602] [Help]
### Summary

I have a website hosted in aws named "memetose" where a user can order for a project to another user. The problem is in "POST" request. The 'post' request is working fine in localhost and even in vercel but not in aws. It shows 500 status code and "VM179:1 Uncaught (in promise) SyntaxError: Unexpected end of JSON input" error. I'm totally clue less about the problem. What could be the problem here? Is it problem about aws or cors policy? Thanks in advance.  
![Screenshot 2023-08-26 221833](https://github.com/vercel/next.js/assets/96897895/5d65578b-2082-43cb-b08c-22b2cfa0660b)
![Screenshot 2023-08-26 222105](https://github.com/vercel/next.js/assets/96897895/f7e140c9-613b-4889-8410-59435441b5de)

**Comment by voidrlm** [comment]
Did you enable CORS for your API Gateway ?

**Reply by voidrlm** [reply]
Read the documentation on how CORS can be enabled. 
https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-cors-console.html
